.github/workflows/publish-ucd.sh

```diff
@@ -1,46 +1,56 @@
 # See publish-ucd.yml
 
-TODAY=`date --iso-8601`
-
 mkdir dist
 
+PUB_DATE=$(date --iso-8601)
+
 cat > dist/sed-readmes.txt << eof
 s/COPY_YEAR/$COPY_YEAR/
-s/PUB_DATE/$TODAY/
+s/PUB_DATE/$PUB_DATE/
 s/PUB_STATUS/draft/
 s/UNI_VER/$UNI_VER/
 s/EMOJI_VER/$EMOJI_VER/
+s/TR10_REV/$TR10_REV/
 s%PUBLIC_EMOJI%Public/draft/emoji%
 s%PUBLIC_UCD%Public/draft/UCD%
 eof
 
-mkdir -p dist/UCD/ucd
-mkdir dist/zipped
-cp -r unicodetools/data/ucd/dev/* dist/UCD/ucd
-rm -r dist/UCD/ucd/Unihan
+mkdir dist/UCD
+cp -R unicodetools/data/ucd/dev dist/UCD/ucd
 mv dist/UCD/ucd/version-ReadMe.txt dist/UCD/ReadMe.txt
-mv dist/UCD/ucd/zipped-ReadMe.txt dist/zipped/ReadMe.txt
+rm -r dist/UCD/ucd/Unihan
+if [ "$MODE" = "Snapshot" ]; then
+    rm -r dist/UCD/ucd/emoji
+fi
 
-if [ $MODE = "Alpha" ]; then
-    mkdir dist/emoji
-    cp unicodetools/data/emoji/dev/* dist/emoji
+if [ "$MODE" = "Alpha" ] || [ "$MODE" = "Beta" ]; then
+    cp -R unicodetools/data/emoji/dev dist/emoji
 
-    mkdir dist/idna
-    cp unicodetools/data/idna/dev/* dist/idna
+    cp -R unicodetools/data/idna/dev dist/idna
 
     mkdir dist/idna2008derived
-    cp unicodetools/data/idna/idna2008derived/Idna2008-$UNI_VER.txt dist/idna2008derived
     cp unicodetools/data/idna/idna2008derived/ReadMe.txt dist/idna2008derived
-else
-    rm -r dist/UCD/ucd/emoji
+    cp unicodetools/data/idna/idna2008derived/Idna2008-$UNI_VER.txt dist/idna2008derived
+
+    if [ "$MODE" = "Beta" ]; then
```

Comment:
Please move this to after the alpha-or-beta block. False economy to nest it here.